editor/src/components/canvas/canvas-strategies/strategies/drag-to-insert-strategy.spec.browser2.tsx

```diff
@@ -26,6 +27,7 @@ async function setupInsertTest(inputCode: string): Promise<EditorRenderResult> {
 
 async function dragFromInsertMenuDivButtonToPoint(
   targetPoint: { x: number; y: number },
+  modifiers: Modifiers,
```

Comment:
I guess we should rename this file too to `drag-to-insert-metastrategy.spec.browser2.tsx`

Reply:
Ha yes! We also should rename the draw-to-insert one too, but I'll do that separately incase someone else is making changes to it